### PR TITLE
initial implementation

### PR DESCRIPTION
**Pull request by prolic**

### CODE REVIEW

**Comment by prolic** [comment]
@codeliner ready for review, let's add some minimal docs and publish 1.0.0-BETA.1

File: src/AmqpCommandConsumerCallback.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        );
+
+        if (false === $data['created_at']) {
+            return DeliveryResult::MSG_REJECT();
+        }
+
+        try {
+            $command = $this->messageFactory->createMessageFromArray($envelope->getType(), $data);
+            $this->commandBus->dispatch($command);
+        } catch (\Exception $e) {
```
Comment (codeliner) [inline src/AmqpCommandConsumerCallback.php]: catch (\Throwable $e) //to catch also new PHP7 errors

Comment (prolic) [inline src/AmqpCommandConsumerCallback.php]: great hint, I need to do the same in HumusAmqp, simply forgot about this new feature.

Comment (prolic) [inline src/AmqpCommandConsumerCallback.php]: added issue here: https://github.com/prolic/HumusAmqp/issues/13

**Comment by prolic** [comment]
@codeliner updated

**Comment by codeliner** [comment]
Merged. great job. cool that we have amqp support now. 

fyi: I'm off for the rest of the day.

**Pull request merged**